RELEASE_NOTES.rst

```diff
@@ -21,7 +21,7 @@
 
 .. towncrier release notes start
 
-Airflow 2.5.3 (2023-03-30)
+Airflow 2.5.3 (2023-03-31)
```

Comment:
I think we should add the fix PR to the change log.. it's a small release so if we can list all relevant PRs it's better

Reply:
Done, thanks for the additional context provided on slack :)